app/docs/introduction/staff-roster.tsx

```diff
@@ -11,37 +12,72 @@ export async function getStaffRoster(): Promise<StaffRosters> {
   const client = await clientPromise;
   const db = client.db("ShadowrunWeb");
 
-  const adminRoster = await db
+  const staffMembers = await db
     .collection("Staff")
-    .find({ staffTitle: "Admin" })
-    .toArray();
-  const moderatorRoster = await db
-    .collection("Staff")
-    .find({ staffTitle: "Moderator" })
+    .find({ staffTitle: { $in: ["Founder", "Admin", "Moderator"] } })
     .toArray();
 
+  const founderRoster = staffMembers.filter((roster) =>
+    roster.staffTitle.includes("Founder")
+  );
+  const adminRoster = staffMembers.filter(
+    (roster) =>
+      roster.staffTitle.includes("Admin") &&
+      !roster.staffTitle.includes("Founder")
+  );
+  const moderatorRoster = staffMembers.filter(
+    (roster) =>
+      roster.staffTitle.includes("Moderator") &&
+      !roster.staffTitle.includes("Admin")
+  );
+
```

Comment:
Nice dude I like the 1 request to the db, then filtering client side. Good practice, instead of 3 calls to the db.